src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -64,13 +64,15 @@ static void usage()
             << "  --device <device path>                    Specify nbd device path\n"
             << "  --read-only                               Map readonly\n"
             << "  --nbds_max <limit>                        Override for module param\n"
+            << "  --single-client                           Forbid other clients write\n"
```

Comment:
Minor: how about just "--exclusive" or "--exclusive-client"? Also, can you add the argument to the "rbd nbd map" CLI as well?